pkg/request/batchstore.go

```diff
@@ -7,36 +7,34 @@ package request
 
 import (
 	"context"
-	"fmt"
 	"sync"
 	"sync/atomic"
-	"time"
 )
 
 type BatchStore struct {
```

Comment:
what about a test that tests this?

Reply:
A unit test for batch store?
I'll add one